src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -867,6 +879,10 @@ private void SeekCompounds(in Entity entity, ref MicrobeAI ai, ref WorldPosition
             {
                 ai.LastSmelledCompoundPosition = null;
                 RunAndTumble(ref ai, ref control, ref position, ref absorber, compounds, speciesActivity, random);
+
+                if (ai.Strain < Constants.MAX_STRAIN_PER_ENTITY * 0.7f)
+                    control.Sprinting = true;
```

Comment:
I could think of two reasons to not sprint for a compound cloud:

1) It's a less efficient use of resources if you're trying to survive compound cloud to compound cloud

2) it might leave you vulnerable if a predator happens by right when you finish your sprint

Maybe this behavior should be behind a check for activity or focus above a certain threshold?

Reply:
If I understood the context properly, then I agree that the AI should not try to run to find compound clouds as running to gather basic resources should be a waste of resources (as long as sprinting is properly balanced in terms of ATP cost).